pure-variants/Dockerfile

```diff
@@ -15,8 +15,9 @@ ONBUILD COPY ./dependencies /tmp/dependencies
 FROM build_${BUILD_TYPE}
 
 USER root
-COPY ./updateSite /tmp/pure-variants
 
+COPY ./updateSite /tmp/pure-variants
+COPY ./license.lic /home/techuser/pure-variants-5/de.license
```

Comment:
The license file should be mounted during runtime, so this line is not needed. 
Do you know if there are different files stored in the `pure-variants-5` directory? In Kubernetes, secrets are mounted as directory (which means that the `/home/techuser/pure-variants-5/de.license` directory is not writable when mounted as read-only). 

I'd prefer a solution where we mount the file to `/inputs/pure-variants/license.lic` and it's copied in the workspace preparation.

Reply:
The `/home/techuser/pure-variants-5/` directory needs to stay writable, so I implemented your suggestion.